### ISSUE TITLE
text is blurry

### ISSUE THREAD

**Issue by Jorropo** [narc0tiq/YARM#177]
Since 0.11.1 the text looks blurry:

![image](https://github.com/user-attachments/assets/c6053de6-335e-4259-b485-408cd91032a0)

If I manually downgrade to 0.11.0 it looks crisp (new save it opening a save with 0.11.1 using 0.11.0 crashes the game instantly):

![image](https://github.com/user-attachments/assets/89fe31d2-0151-49ab-bbe7-d981e7a2ddea)

I'm not exactly sure why it looks blurry.

If I go pixel peeping on the old version the text "glowing" on the neighboring pixels (anti aliasing or subpixel rendering ¿ that where my knowledge stop).
The new version's text is identical but the black outline does not "glow", the text itself still glows but on the outline. So I guess this bug report could be rephrased: `the black text outline added in 0.11.1 is not antialiased`
However there are very few pixels to interpret and I don't know if that subjectively looks blurry since the letters are fatter.

I don't know how this text is rendered on screen, I guess could be a factorio if that using primitives provided by factorio.

**Comment by wchristian** [comment]
as a very first sanity check: do you have some strong sharpening enable?

since uh, i can't see any of the blurriness you mean, whatsoever. both texts have about the same smoothness to me and the only difference is an added outline

**Comment by Jorropo** [comment]
I didn't thought about the display settings, after checking I don't have sharpening enabled but I had extremely low brightness. Raising the brightness fixes the issue but I don't like it.

I've tried taking a photo but the eyes are much better at picking up green text on green grass contrast without the black outline than my phone even with HDR, make sense why that was added.

Closing as it's a brightness tradeoff.
